src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveSamlEntityCommand.php

```diff
@@ -458,21 +464,22 @@ public function setPastedMetadata($pastedMetadata)
     }
 
     /**
-     * @return string
+     * @param $acsLocations
      */
-    public function getAcsLocation(): ?string
+    public function setAcsLocations($acsLocations)
```

Comment:
I'd still love to see you fix this. Don't worry about inconsistencies between new and old code. The old annotations where only added to aid PHP Storm for code completion 

`public function setAcsLocations(array $acsLocations): void`

This feedback also applies to some of the other new methods you created.